worldengine/model/world.py

```diff
@@ -132,11 +202,9 @@ def _from_protobuf_quantiles(p_quantiles):
 
     @staticmethod
     def _from_protobuf_matrix_with_quantiles(p_matrix):
```

Comment:
You are changing almost every line anyway, the PR wouldn't really get much bigger, I think.^^